client/src/components/Auth/Login.js

```diff
@@ -58,7 +45,7 @@ const Login = (props) => {
             id="submit"
             onClick={(e) => {
               e.preventDefault();
-              props.submit(state.username, state.email, state.password);
+              props.submit(state.username, state.password);
```

Comment:
I noticed you removed the email, I am not sure how exactly this works but I think it would be nice to let a user log in with either their email or username.